test/inject.test.js

```diff
@@ -222,6 +222,72 @@ test('inject get request - reply stream', t => {
   })
 })
 
+test('inject promisify - waiting for ready event', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  const payload = { hello: 'world' }
+
+  fastify.get('/', (req, reply) => {
+    reply.send(payload)
+  })
+
+  fastify.inject({
+    method: 'GET',
+    url: '/'
+  })
+    .then(res => {
+      t.strictEqual(res.statusCode, 200)
+    })
```

Comment:
is this passing linting? can you format this better?

Reply:
Yeah, travis says that.

> can you format this better?

😃

Reply:
this is still not nice to read